### PR TITLE
Support automated runs in the web version.

### PR DESCRIPTION
**Pull request by ulan**
This adds window.automated object that allows to run the benchmark in Catapult/Telemetry framework by evaluating JS.

### CODE REVIEW

File: src/bootstrap.js [hunk 1]
```diff
@@ -46,8 +46,12 @@ function reset() {
       } else {
         const benchmark = benchmarks[index];
         text += `<td class="benchmark-name">`;
-        text += `<a href="https://github.com/v8/web-tooling-benchmark/blob/master/docs/in-depth.md#${benchmark.name}" target="_blank">${benchmark.name}</a></td>`;
-        text += `<td class="result" id="results-cell-${benchmark.name}">&mdash;</td>`;
```
Comment (bmeurer) [inline src/bootstrap.js]: These changes seem unrelated, can you undo them?

File: src/bootstrap.js [hunk 2]
```diff
@@ -129,10 +148,13 @@ suite.on("complete", event => {
 });
 
 suite.on("error", event => {
+  window.automated.completed = true;
   const benchmark = event.target;
   const error = benchmark.error;
   const name = benchmark.name;
-  document.body.innerHTML = `<h1>ERROR</h1><p>Encountered errors during execution of ${name} test. Refusing to run a partial benchmark suite.</p><pre>${error.stack}</pre>`;
```
Comment (bmeurer) [inline src/bootstrap.js]: Also undo these unrelated changes.

File: src/bootstrap.js (hunk 1 above)
Comment (mathiasbynens) [reply src/bootstrap.js]: It might've been `prettier` making these changes.

File: src/bootstrap.js (hunk 2 above)
Comment (mathiasbynens) [reply src/bootstrap.js]: Similar to the above, I think this is just prettier doing its thing. In that case, we should keep the changes.
